docs/rules/no-node-access.md

```diff
@@ -45,6 +45,17 @@ const signinModal = getByLabelText('Sign In');
 within(signinModal).getByPlaceholderText('Username');
 ```
 
+```js
+import { screen } from '${testingFramework}';
```

Comment:
Can you put an actual testing library module in this import?

Reply:
Ah, this is a good catch. This only related to `react`, so I'll put that there.

Reply:
Done!